features/font-variant-position.yml.dist

```diff
@@ -2,36 +2,24 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 status:
-  baseline: low
-  baseline_low_date: 2023-09-15
+  baseline: false
```

Comment:
`font-variant-position` moved backwards due to Chrome's partial implementation. See https://github.com/mdn/browser-compat-data/issues/24136 for details and why the per-key statuses will be (hopefully temporarily) wrong.